src/Http/Controllers/Api/Geojson/ImageAnnotationsController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Biigle\Modules\Geo\Http\Controllers\Api\Geojson;
+
+use DB;
+use Biigle\Image;
+use GeoJson\Feature\{Feature, FeatureCollection};
+use GeoJson\Geometry\Point;
+use Biigle\Http\Controllers\Api\Controller;
+use League\Flysystem\FileNotFoundException;
+
+class ImageAnnotationsController extends Controller{
+
+  public function index($id)
+  {
+    $image = Image::findOrFail($id);
+    $this->authorize('access', $image);
+
+    $metadata = $image->metadata;
+    $image_width_m = 2 * (float)$image->metadata['distance_to_ground'];
+    $result = $image->annotations->map(function($annotation) use($image, $metadata,$image_width_m){
+      $annotation_points = array_slice($annotation->points, 0, 2);
+      $image_center = [($image->width)/2, ($image->height)/2];
+
+      # Finding Annotation Point with respect to center of the image.
+      $annotation_point = [($annotation_points[0] - $image_center[0]), ($image_center[1] - $annotation_points[1])];
+
+      # X and Y Coordinate Rotations According to Yaw
+      $rotated_X = $annotation_point[0] * cos((Float)$metadata['yaw']) + $annotation_point[1] * sin((Float)$metadata['yaw']);
+      $rotated_Y = $annotation_point[0] * sin((Float)$metadata['yaw']) + $annotation_point[1] * cos((Float)$metadata['yaw']);
```

Comment:
According to [this answer](https://stackoverflow.com/a/34374437/1796523) the rotation formula of a point `p` around a point `o` by angle `angle` (in radians) is:

```
qx = ox + cos(angle) * (px - ox) - sin(angle) * (py - oy)
qy = oy + sin(angle) * (px - ox) + cos(angle) * (py - oy)
```

The initial addition of `ox` and `oy` can be discarded in this case, as we want to get the offset from the image center and not the actual new coordinate. The yaw is given in degrees which need to be converted with [`deg2rad`](https://www.php.net/manual/en/function.deg2rad.php). The term for the x value should be a subtraction instead of a n addition.